frontend/src/App.tsx

```diff
@@ -42,7 +42,7 @@ function App() {
                 if (!response.data) {
                     navigate("/login");
                 } else {
-                    navigate("/");
+                    navigate(`/${response.data.id}`);
```

Comment:
Why do you want to navigate to the ID of the user every time the user information is loaded? Sometimes you maybe might want to load user information without navigating somewhere. Also you could name this path i. e. `/profile`.